### ISSUE TITLE
Why cycle loss use L1 loss？

### ISSUE THREAD

**Issue by CrystalWong2** [junyanz/pytorch-CycleGAN-and-pix2pix#853]
I wonder why cycleloss use L1 lossfunction. I'm new in CV. I think maybe people almost like to use MSE or something else. Did you try to change the cycle loss to other loss function?  If you try, is it the best? Just want to know why,thank you

**Comment by junyanz** [comment]
It does not matter a lot. You can use MSE, or other kinds of Lp norm loss, or a hybrid of L1/L2 loss. We found that L1 loss produces slightly sharper results while L2 loss tends to produce more blurry results.

**Comment by kingstarcraft** [comment]
@CrystalWong2 Hi, do you have tried smooth-l1.

**Comment by junyanz** [comment]
Smooth-l1 loss might also work. Feel free to try it.
